docs/source/guides/build.rst

```diff
@@ -52,66 +68,55 @@ to the directory of your choice.
   save some time and bandwidth. If you want to get a full commit history, you
   can omit this option.
 
-After it, you need to run the development environment. Run the
+When it is done, you need to run the development environment. Run the
 ``scripts/run-iroha-dev.sh`` script:
 
 .. code-block:: shell
 
   bash scripts/run-iroha-dev.sh
 
 .. hint:: Please make sure that Docker is running before executing the script.
-  macOS users could find a Docker icon in system tray, Linux user could use
+  MacOS users could find a Docker icon in system tray, Linux users can use
   ``systemctl start docker``
 
-After you execute this script, following things happen:
+After you execute this script, the following things will happen:
 
-1. The script checks if you don't have containers with Iroha already running.
+#. The script will check whether you have containers with Iroha already running.
 Successful completion finishes with the new container shell.
 
-2. The script will download ``hyperledger/iroha:develop-build`` and ``postgres`` images.
+#. The script will download ``hyperledger/iroha:develop-build`` and ``postgres`` images.
 ``hyperledger/iroha:develop-build`` image contains all development dependencies and is
 based on top of ``ubuntu:16.04``. ``postgres`` image is required for starting
 and running Iroha.
 
-3. Two containers are created and launched.
+#. Two containers are created and launched.
 
-4. The user is attached to the interactive environment for development and
+#. The user is attached to the interactive environment for development and
 testing with ``iroha`` folder mounted from the host machine. Iroha folder
 is mounted to ``/opt/iroha`` in Docker container.
 
-Now your are ready to build Iroha! Please go to `Building Iroha` section.
+Now your are ready to build Iroha! Please go directly to `Building Iroha <#build-process>`_ section.
+
+.. _linux-pre:
 
 Linux
 ^^^^^
 
-Boost
-"""""
-
-Iroha requires Boost of at least 1.65 version.
-To install Boost libraries (``libboost-all-dev``), use `current release
-<http://www.boost.org/users/download/>`_ from Boost webpage. The only
-dependencies are thread, system and filesystem, so use
-``./bootstrap.sh --with-libraries=thread,system,filesystem`` when you are building
-the project.
-
-Other Dependencies
-""""""""""""""""""
-
-To build Iroha, you need following packages:
+To build Iroha, you will need the following packages:
 
 ``build-essential`` ``automake`` ``libtool`` ``libssl-dev`` ``zlib1g-dev``
```

Comment:
Pretty sure that this list can be reduced to `build-essential, git, cmake, ninja-build, curl, unzip, tar`